yt_dlp/extractor/bannedvideo.py

```diff
@@ -0,0 +1,235 @@
+            __typename
+        }
+        sale {
+            _id
+            text
+            __typename
+        }
+        __typename
+    }
+}
+fragment DisplayVideoFields on Video {
+    _id
+    title
+    summary
+    playCount
+    largeImage
+    embedUrl
+    published
+    videoDuration
+    channel {
+        _id
+        title
+        avatar
+        __typename
+    }
+    createdAt
+    __typename
+}'''
+    _GRAPHQL_GETCOMMENTS_QUERY = '''
+query GetVideoComments($id: String!, $limit: Float, $offset: Float) {
+    getVideoComments(id: $id, limit: $limit, offset: $offset) {
+        ...VideoComment
+        replyCount
+        __typename
+    }
+}
+fragment VideoComment on Comment {
+    _id
+    content
+    liked
+    user {
+        _id
+        username
+        __typename
+    }
+    voteCount {
+        positive
+        __typename
+    }
+    linkedUser {
+        _id
+        username
+        __typename
+    }
+    createdAt
+    __typename
+}'''
+    _GRAPHQL_GETCOMMENTSREPLIES_QUERY = '''
+query GetCommentReplies($id: String!, $limit: Float, $offset: Float) {
+    getCommentReplies(id: $id, limit: $limit, offset: $offset) {
+        ...VideoComment
+        replyTo {
+            _id
+            __typename
+        }
+        __typename
+    }
+}
+fragment VideoComment on Comment {
+    _id
+    content
+    liked
+    user {
+        _id
+        username
+        __typename
+    }
+    voteCount {
+        positive
+        __typename
+    }
+    linkedUser {
+        _id
+        username
+        __typename
+    }
+    createdAt
+    __typename
+}'''
+    _VALID_URL = r'https?://(?:www\.)?banned\.video/watch\?id=(?P<id>[0-f]{24})'
+    _TEST = {
+        'url': 'https://banned.video/watch?id=5e7a859644e02200c6ef5f11',
+        'md5': '14b6e81d41beaaee2215cd75c6ed56e4',
+        'info_dict': {
+            'id': '5e7a859644e02200c6ef5f11',
+            'ext': 'mp4',
+            'title': 'China Discovers Origin of Corona Virus: Issues Emergency Statement',
+            'thumbnail': r're:^https?://(?:www\.)?assets\.infowarsmedia.com/images/',
+            'description': 'The Chinese Communist Party Official Spokesperson At the Ministry of Truth Released Their Statement Exclusively To Alex Jones and Infowars.com',
+            'upload_date': '20200324',
+            'timestamp': 1585087895,
+        }
+    }
+
+    def _real_extract(self, url):
+        video_id = self._match_id(url)
+        video_info = try_get(
+            self._download_json(
+                self._GRAPHQL_API,
+                video_id,
+                headers=self._GRAPHQL_HEADERS,
+                data=json.dumps({
+                    'variables': {
+                        'id': video_id
+                    },
+                    'operationName': 'GetVideo',
+                    'query': self._GRAPHQL_GETVIDEO_QUERY
+                }).encode('utf8')),
+            lambda x: x['data']['getVideo'])
+        video_comments = try_get(
```

Comment:
Right now only the default value for comment limit is requested. But if there are more they won't be requested.